scripts/ci/run_ci.sh

```diff
@@ -138,10 +139,12 @@ else
 fi
 
 # Build Docs on matrix 5 in a pull request
-if [ "${MATRIX}" = "${DOC_MATRIX}" -a -n "${PULL_REQUEST}" ]; then
-	build_docs
-	./scripts/ci/check-compliance.py --commits ${COMMIT_RANGE} || true;
-fi
+# Skipping those in main CI job and moving them to a new job
+# that reports directly into the PR.
+#if [ "${MATRIX}" = "${DOC_MATRIX}" -a -n "${PULL_REQUEST}" ]; then
+#	build_docs
```

Comment:
Is this correct, seems like lines 144-147 shouldn't start with '#'?

Reply:
yes, this is the intent, maybe I should just remove the lines completely.